### ISSUE TITLE
After dismissing the "Support your favorite sites" banner, focus is not returned to the active tab's content area

### ISSUE THREAD

**Issue by bsclifton** [brave/browser-laptop#8742]
This is likely a more broad issue (ex: affecting all notifications and the accepting of their options).  

But issue as reported:
Being presented with the "Are you ready to support your favorite sites".  You choose "no" and focus isn't sent back to the active tab's content area

**Comment by bsclifton** [comment]
Related https://github.com/brave/browser-laptop/issues/7666

**Comment by rederekt** [comment]
This behavior is still present in v0.23.104 on MacOS. My test is opening a window, running alert() from the console, dismissing the popup, then trying to scroll by pressing the arrow keys. This doesn't work until I click on the window again.

I can take a look in more detail tomorrow.

**Comment by rederekt** [comment]
Strange...whenever a banner like "Allow Google to access your location" pops up, I see the following warnings in the console: 
Aug 29 00:49:03  Brave Helper[18597] <Warning>: CGSConnectionByID: 0 is not a valid connection ID.
Aug 29 00:49:03  Brave Helper[18597] <Warning>: Invalid Connection ID 0

I think this is macOS-exclusive, but perhaps something to keep in mind.

Any insight into what these are?

**Comment by petemill** [comment]
@rederekt `webviewActions.setWebviewFocused()`

**Comment by rederekt** [comment]
Thanks @petemill ! That's one of the few I hadn't tried 😅 

Should have a PR fix submitted soon

**Comment by rederekt** [comment]
PR submitted (#15130) 😄

**Comment by bsclifton** [comment]
Closing as we move to a new code-base (where Brave Payments is being reworked and will now be known as Brave Rewards)
